packages/database/prisma/migrations/20230921110737_initial/migration.sql

```diff
@@ -0,0 +1,271 @@
+    `sampleSize` INTEGER NOT NULL,
+    `totalRecruitmentToDate` VARCHAR(191) NOT NULL,
+    `plannedOpeningDate` DATETIME(3) NOT NULL,
+    `plannedClosureDate` DATETIME(3) NOT NULL,
+    `actualOpeningDate` DATETIME(3) NOT NULL,
+    `actualClosureDate` DATETIME(3) NOT NULL,
+    `expectedReopenDate` DATETIME(3) NOT NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+    `isDeleted` BOOLEAN NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `StudyFunder` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `studyId` INTEGER NOT NULL,
+    `organisationId` INTEGER NOT NULL,
+    `grantCode` VARCHAR(191) NOT NULL,
+    `fundingStreamName` VARCHAR(191) NOT NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+    `isDeleted` BOOLEAN NULL DEFAULT false,
+
+    UNIQUE INDEX `StudyFunder_studyId_organisationId_key`(`studyId`, `organisationId`),
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `Assessment` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `studyId` INTEGER NOT NULL,
+    `statusId` INTEGER NOT NULL,
+    `createdById` INTEGER NOT NULL,
+    `createdAt` DATETIME(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
+    `updatedAt` DATETIME(3) NOT NULL,
+    `isDeleted` BOOLEAN NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `AssessmentFurtherInformation` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `assessmentId` INTEGER NOT NULL,
+    `furtherInformationId` INTEGER NOT NULL,
+    `furtherInformationText` TEXT NOT NULL,
+    `isDeleted` BOOLEAN NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `SysRefRole` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(191) NOT NULL,
+    `description` TEXT NOT NULL,
+    `isDeleted` BOOLEAN NOT NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `SysRefAssessmentStatus` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(191) NOT NULL,
+    `description` TEXT NOT NULL,
+    `isDeleted` BOOLEAN NOT NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `SysRefAssessmentFurtherInformation` (
+    `id` INTEGER NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(191) NOT NULL,
+    `description` TEXT NOT NULL,
+    `sortOrder` INTEGER NOT NULL,
+    `isDeleted` BOOLEAN NOT NULL DEFAULT false,
+
+    PRIMARY KEY (`id`)
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+-- CreateTable
+CREATE TABLE `SysRefOrganisationRole` (
```

Comment:
This is the only SysRef we'll seed as part of the ingest. The others we'll need to seed via a migration in our app.